web/src/pages/Route/components/Step1/MatchingRulesView.tsx

```diff
@@ -217,6 +220,7 @@ const MatchingRulesView: React.FC<RouteModule.Step1PassProps> = ({
               <Option value="http">{formatMessage({ id: 'page.route.httpRequestHeader' })}</Option>
               <Option value="arg">{formatMessage({ id: 'page.route.requestParameter' })}</Option>
               <Option value="cookie">Cookie</Option>
+              <Option value="buildin">{formatMessage({ id: 'page.route.buildinParameter' })}</Option>
```

Comment:
need @liuxiran and @tzssangglass 's confirmation.

[1] https://apisix.apache.org/docs/apisix/admin-api#request-body-parameters
[2] https://github.com/api7/lua-resty-expr

Reply:
cookie is builtin Nginx variables.

"cookie_x_foo" means to use the key with "x_foo" cookie.

Reply:
Just to make sure, if there has an Option called `buildin` here?

Reply:
I think `buildin` is required, see: http://nginx.org/en/docs/varindex.html

Reply:
ok